cfme/tests/intelligence/reports/test_validate_chargeback_report.py

```diff
@@ -118,6 +118,15 @@ def assign_compute_default_rate(provider):
     enterprise.computeassign()
     logger.info('ASSIGNING DEFAULT COMPUTE RATE')
 
+    yield
+
```

Comment:
**OPTIONAL:** please add a comment briefly explaining the cleanup to ease future reading